examples/js/controls/OrbitControls.js

```diff
@@ -302,7 +325,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 		return function panLeft( distance, objectMatrix ) {
 
 			v.setFromMatrixColumn( objectMatrix, 0 ); // get X column of objectMatrix
-			v.multiplyScalar( - distance );
+			v.multiplyScalar( - distance * scope.panSpeed );
```

Comment:
What do you think the units of panSpeed are?

Reply:
Thanks for the review. Good question... `panSpeed` is a coefficient but one could formulate it as the number of GL units added to a displacement of 1 GL unit (GLunit/GLunit..) in the time elapsed since the previous `update` call ?

Reply:
That would be unit-less -- a "pan speed factor". But I wouldn't worry about the nomenclature at this point.